portal-kernel/src/com/liferay/portal/kernel/feature/flag/constants/FeatureFlagConstants.java

```diff
@@ -17,6 +17,11 @@ public class FeatureFlagConstants {
 
 	public static final String FEATURE_FLAG = "feature.flag";
 
+	public static final String PREFERENCE_DEPRECATION_PROCESSED_KEY =
+		"deprecation.feature.flag.processed";
+
+	public static final String PREFERENCE_NAMESPACE = FEATURE_FLAG;
```

Comment:
@thiago-buarqque This key PREFERENCE_NAMESPACE is a duplication of FEATURE_FLAG, no need to have it

Reply:
Updated it, thanks @tinatian